workitem/undoable_wi_repo.go

```diff
@@ -33,12 +33,12 @@ type UndoableWorkItemRepository struct {
 }
 
 // Load implements application.WorkItemRepository
-func (r *UndoableWorkItemRepository) Load(ctx context.Context, ID string) (*app.WorkItem, error) {
```

Comment:
yes, I'd be happy if we could get rid of that file, too (it's a bit confusing at first sight).